Add hardcode Authfile for windows and mac (#944)
containers/image: docs/containers-auth.json.5.md

> A credentials file stored at `${XDG_RUNTIME_DIR}/containers/auth.json` in
> json format used to authenticate against container image registries.
> For windows and darwin systems the credential file is stored at `$HOME/.config/containers/auth.json`

Reviewer: Something like
```suggestion
A credentials file in JSON format used to authenticate against container image registries.
On Linux it is stored at `${XDG_RUNTIME_DIR}/containers/auth.json`;
on Windows and macOS, at `$HOME/.config/containers/auth.json`
```
